tests/Services/ShopSessionTest.php

```diff
@@ -122,21 +122,23 @@ public function testCanForget()
     public function testSameSiteCookie()
     {
         $incompatibleUserAgents = [
+            'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.78 Safari/537.36',
             'Mozilla/5.0 (iPhone; CPU iPhone OS 12_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) GSA/87.0.279142407 Mobile/15E148 Safari/605.1',
             'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.2 Safari/605.1.15',
             'Mozilla/5.0 (Linux; U; Android 7.0; en-US; SM-G935F Build/NRD90M) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 UCBrowser/11.3.8.976 U3/0.8.0 Mobile Safari/534.30',
             'UCWEB/2.0 (Java; U; MIDP-2.0; en-US; generic) U2/1.0.0 UCBrowser/9.5.0.449 U2/1.0.0 Mobile',
-            'Mozilla/5.0 (Linux; U; Android 9; zh-CN; Nokia X7 Build/PPR1.180610.011) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.108 UCBrowser/12.14.0.1020 Mobile Safari/537.36',
```

Comment:
Any particular reason this one was removed?

Reply:
This wasn't removed. It got moved to the compatible list. This is a user agent for UBrowser 12.14, not Chrome 57.

Reply:
By the way, this is also the reason why I've changed the browser detection from `$agent->is()` to `$agent->browser() ==`. This particular user agent returns true for `$agent->is('chrome')` while `$agent->browser()` returns `UCBrowser`, which is why the browser version ends up (confusingly during my debugging) returning `12.14` instead of `57.0`

Reply:
Interesting.

Wow, this change is a real headache, and I think a large part of the internet is going to 'break' when Chrome 80 rolls out.